core/pom.xml

```diff
@@ -267,6 +267,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.1.1</version>
+        <executions>
+          <execution>
+            <id>generate-dependency-list</id>
+            <goals>
+              <goal>list</goal>
+            </goals>
+            <phase>prepare-package</phase>
```

Comment:
Not that it makes any difference in practice, but `generate-resources` feels like a more logical choice.

@tomekl007 did we discuss that already in the original ticket?

Reply:
I don't recall a discussion about this topic.

Reply:
done